Black-Market/Black-Market/Models/UserDataResponse.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UserResponse.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+
+import Foundation
+
+struct UserDataResponse: Codable {
+    let accessToken: String
+    let refreshToken: String
+    let user: UserModel
+    
+    
+    enum CodingKeys: String, CodingKey {
+        case accessToken = "access_token"
+        case refreshToken = "refresh_token"
+        case user
+    }
+}
+
+struct UserModel: Codable {
+    let id: Int
+    let email: String
+    let name: String
+    let profilePicture: String
+    let birthDate: String
```

Comment:
similar here, could this be a `Date` object instead? a `dateDecodingStrategy` can be set to the JSONDecoder if the format is the same for all dates.

Reply:
Done